JuliaInterface/gap/BindCFunction.gi

```diff
@@ -10,7 +10,7 @@
 
 InstallGlobalFunction( JuliaBindCFunction,
   function( julia_name, nr_args, arg_names )
-    local cfunction_call_string, i, cfunc;
+    local cfunction_call_string, i, cfunc, function_ptr, arg_string;
```

Comment:
Why does one need to specify `nr_args`? Isn't that always equal to `Length(arg_names)`?